src/server/server.ts

```diff
@@ -23,6 +23,11 @@ app.get('/api/cars', ({ query }: Request<{}, {}, {}, Query>, res) => {
   const { tag } = query;
   const carsByTag = getCarsByTag(tag, cars);
 
+  if (carsByTag.length === 0) {
+    res.json(["No Results"]);
```

Comment:
what's the existing response before this @kml1990 ? I think an empty array would be the norm

Reply:
Agree @dalyr95 👍  Empty array would definitely be better here 👍 Will change that 👍